deployments/dev/docker-compose.vs.debug.yml

```diff
@@ -8,10 +8,7 @@ services:
       dockerfile: Dockerfile
       args:
         source: ${DOCKER_BUILD_SOURCE}
-    depends_on: 
```

Comment:
Graphite was only for tesing the addon we've developed, there's no need to run it in order to run/debug Tweek locally